modules/all-modules/pom.xml

```diff
@@ -118,11 +118,6 @@
         </dependency>
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-file-extension-common</artifactId>
```

Comment:
Not needed for this PR, but we need to decide wether to put file common, as it is not a module... but is not a SDK extension neither